web_src/js/features/codeeditor.js

```diff
@@ -136,7 +136,7 @@ function togglePreviewDisplay(previewable) {
   if (!previewTab) return;
 
   if (previewable) {
-    const newUrl = (previewTab.getAttribute('data-url') || '').replace(/(.*)\/.*/i, `$1/markup`);
```

Comment:
Was the `i` removed as this regex does not care about case at all?

Reply:
Look at https://github.com/go-gitea/gitea/pull/24361#issuecomment-1523865177